libindex/libindex_integration_test.go

```diff
@@ -202,8 +202,8 @@ func checkEqual(ctx context.Context, t *testing.T, tc testcase, lib *Libindex, i
 		cmp.FilterPath(func(p cmp.Path) bool {
 			s := p.Index(-3)
 			m := p.Last().String()
-			return s.Type() == reflect.TypeOf((*claircore.Package)(nil)) &&
-				(m == ".RepositoryHint" || m == ".PackageDB")
+			return m == ".Files" || (s.Type() == reflect.TypeOf((*claircore.Package)(nil)) &&
```

Comment:
This is a weird side-effect of excluding `Files` from the json output, `Index()` will return:
```
Files: map[string]claircore.File{}
```
And `IndexReport()` will return:
```
Files: nil
```
As it's been unmarshaled from the DB blob